src/SDKs/EventGrid/DataPlane/Microsoft.Azure.EventGrid/Customization/ICustomEventTypeMapper.cs

```diff
@@ -15,6 +15,6 @@ public interface ICustomEventTypeMapper
 
         bool TryGetCustomEventMapping(string eventType, out Type eventDataType);
 
-        IReadOnlyList<KeyValuePair<string, Type>> GetAllCustomEventMappings();
+        IEnumerable<KeyValuePair<string, Type>> ListAllCustomEventMappings();
```

Comment:
Potentially a breaking change requiring major version bump. @shahabhijeet could you take a look

Reply:
One point I want to add is that we haven't yet released a NuGet package that contains the new EventGridSubscriber functionality (that was introduced in the last PR), that's why I haven't updated the package release notes, plus the reason behind this contract change.